go/database/connection.go

```diff
@@ -2,14 +2,20 @@ package database
 
 import (
 	"database/sql"
+	"os"
 
 	"github.com/inverse-inc/packetfence/go/pfconfigdriver"
 )
 
 func Connect(user, pass, host, port, dbname string) *sql.DB {
 	var where string
 	if host == "localhost" {
-		where = "unix(/var/lib/mysql/mysql.sock)"
+		if _, err := os.Stat("/etc/debian_version"); err == nil {
```

Comment:
It would be good to consolidate this function with the one in go/db/db.go to prevent duplication

Reply:
Do we want to do it in this PR ?

Reply:
Since we're already changing stuff in it, I'd say to put it in this PR

Reply:
Ok done